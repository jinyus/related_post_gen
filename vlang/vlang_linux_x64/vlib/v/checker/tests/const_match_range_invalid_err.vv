const end = 10

const start = 1

match 5 {
	end...start {
		println(start)
	}
	else {}
}

a := match 5 {
	end...start {
		4
	}
	else {
		3
	}
}
println(a)
