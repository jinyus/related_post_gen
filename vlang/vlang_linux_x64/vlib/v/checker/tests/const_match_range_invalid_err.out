vlib/v/checker/tests/const_match_range_invalid_err.vv:6:2: error: the start value `10` should be lower than the end value `1`
    4 | 
    5 | match 5 {
    6 |     end...start {
      |     ~~~~~~~~~~~
    7 |         println(start)
    8 |     }
vlib/v/checker/tests/const_match_range_invalid_err.vv:13:2: error: the start value `10` should be lower than the end value `1`
   11 | 
   12 | a := match 5 {
   13 |     end...start {
      |     ~~~~~~~~~~~
   14 |         4
   15 |     }
